deploy.cmd

```diff
@@ -65,6 +65,13 @@ IF /I "%IN_PLACE_DEPLOYMENT%" NEQ "1" (
   IF !ERRORLEVEL! NEQ 0 goto error
 )
 
+:: 2. Check if WebConfig File exists
+IF EXIST %DEPLOYMENT_SOURCE%/web.config (
+  cp %DEPLOYMENT_SOURCE%/web.config %DEPLOYMENT_TARGET%/web.config
```

Comment:
I agree with @khalidabuhakmeh IIS can handle subsequent `web.config`

And i don't think is good doing a `loop` to check which folders have a `web.config` or maybe is good. Idunno.